src/test/java/org/datadog/jenkins/plugins/datadog/publishers/DatadogComputerPublisherTestWrapper.java

```diff
@@ -0,0 +1,15 @@
+package org.datadog.jenkins.plugins.datadog.publishers;
```

Comment:
An alternative way to stub the client would be to use PowerMock as in: https://github.com/jenkinsci/datadog-plugin/pull/74, which means we wouldn't need this file and to add a wrapper `getDatadogClient() `method.